Tribler/Core/Session.py

```diff
@@ -152,19 +150,39 @@ def set_and_create_dir(dirname, setter, default_dir):
 
         self.autoload_discovery = autoload_discovery
 
-    def prestart(self):
-        """ Pre-starts the session. We check the currently version and upgrades if needed
-        before we start everything else.
+    def initialize_database(self):
+        """
+        Initializes the SQLite database.
         """
-        # initialize the database
         self.sqlite_db = SQLiteCacheDB(self)
         self.sqlite_db.initialize()
         self.sqlite_db.initial_begin()
 
-        # check and upgrade
-        upgrader = TriblerUpgrader(self, self.sqlite_db)
-        upgrader.check_and_upgrade()
-        return upgrader
```

Comment:
These are just wrappers for the Upgrade object, why aren't you just using these directly instead?